website/project/views/node.py

```diff
@@ -999,12 +1003,13 @@ def get_folder_pointers(auth, node, **kwargs):
 
 @must_be_contributor_or_public
 def get_forks(auth, node, **kwargs):
-    return _render_nodes(nodes=node.forks, auth=auth)
+    fork_list = sorted(node.forks, key=lambda fork: fork.forked_date, reverse=True)
```

Comment:
Is it possible to use modm to sort the foreign lists instead of sorting in Python?